### ISSUE TITLE
panics by send on closed channel

### ISSUE THREAD

**Issue by KEINOS** [KEINOS/go-getkeypushed#5]
Using v1.1.0 and calling `getkbd.Pushed()` several times panics.

```
panic: send on closed channel
goroutine 16 [running]:
github.com/KEINOS/go-getkeypushed/funcs.CheckTTY(0xc000245860, 0xc0003289c0)
        /go/pkg/mod/github.com/!k!e!i!n!o!s/go-getkeypushed@v1.1.0/funcs/CheckTTY.go:25 +0x145
created by github.com/KEINOS/go-getkeypushed/funcs.GetInput
        /go/pkg/mod/github.com/!k!e!i!n!o!s/go-getkeypushed@v1.1.0/funcs/GetInput.go:30 +0x13e
exit status 2
```

Replication code upcoming.

**Comment by KEINOS** [comment]
2021/05/23 Can't reproduce the panic via testing ...

**Comment by KEINOS** [comment]
May be fixed in PR #6
